packages/data-stores/src/user/controls.ts

```diff
@@ -1,8 +1,13 @@
+/**
+ * External dependencies
+ */
+import wpcomRequest, { WpcomClientCredentials } from 'wpcom-proxy-request';
+
 /**
  * Internal dependencies
  */
-import { wpcomRequest, WpcomClientCredentials } from '../utils';
 import { CreateAccountAction } from './types';
+import '../utils';
```

Comment:
Very much do not like this 😬 

```suggestion
import { requestAllBlogsAccess } from '../utils';
```

Reply:
Can't argue with that :grimacing: I'll point out thatI'm not strictly introducing this here -- it's been like this before, if a bit less visibly so :sweat_smile:

Reply:
(Happy to change that though.)